app/utils/themeUtils.js

```diff
@@ -3,15 +3,17 @@ import imageLogoDarkMode from 'app/assets/logo.png';
 import bannerLightMode from 'app/assets/om-abakus-banner.png';
 import bannerDarkMode from 'app/assets/om-abakus-banner-dark-mode.png';
 
+const isClientSide = () => typeof window !== 'undefined' && document;
+
 export const applySelectedTheme = theme => {
-  if (typeof window !== 'undefined') {
+  if (isClientSide()) {
     document.documentElement.setAttribute('data-theme', theme);
```

Comment:
To make this work with ssr you should add this attr during ssr too

Here:
https://github.com/webkom/lego-webapp/blob/2c57fdfef169e641f491f8ae3e5cb1af3e7d2ac5/server/pageRenderer.js#L60